scripts/build-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+# A script for building binary releases for various platforms.
+
+if [[ "$0" != "./scripts/build-release.sh" ]]; then
+  1>&2 echo "Script must be run at root of filesystem"
+  exit 1
+fi
+
+VERSION="$1"
+
+if [[ "$VERSION" != v* ]]; then
+  1>&2 echo "No version specified as argument"
+  exit 1
+fi
+
+mkdir -p bin
+rm -f bin/*.zip
+rm -f bin/*.tar.gz
+
+function build {
+  TEMP_DIR="$( mktemp -d )"
+
+  export CGO_ENABLED=0
+
+  BIN_NAME="log4jscanner"
+  if [[ "$1" == "windows" ]]; then
+    BIN_NAME="log4jscanner.exe"
+  fi
+
+  GOOS="$1" GOARCH="$2" go build -o "${TEMP_DIR}/log4jscanner/${BIN_NAME}"
```

Comment:
`log4jscanner` subdir doesnt exist: Is this okay (go build will create) or should it be mkdir'd earlier?

Reply:
Go will create it :)